atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolImpl.java

```diff
@@ -392,6 +392,8 @@ void runOneTimeStartupChecks() {
 
             try {
                 V response = runWithPooledResourceRecordingMetrics(hostPool, req.getFunction());
+                log.info("Added host {} back into the pool after receiving a successful response",
```

Comment:
nit: it's a tiny duplication, but did you consider factoring out a method to hold both the log statement and the `blacklist.remove` call?

Reply:
yeah, I was in between. The reason why I choose this was, if I extract it to a function it can only be used for this specific case (removing the host from blacklist after receiving a successful response, couldn't be used if you are adding the host back to pool for some other reason because of the specific log message); so I should have named it to something like removeFromBlacklistAfterResponse(). But yeah, I was in between.